examples/type_conversion.rs

```diff
@@ -0,0 +1,97 @@
+use approx::assert_abs_diff_eq;
+use ndarray::prelude::*;
+use std::convert::TryFrom;
+
+#[cfg(feature = "approx")]
+fn main() {
+    // Converting an array from one datatype to another is implemented with the
+    // `ArrayBase::mapv()` function. We pass a closure that is applied to each
+    // element independently. This allows for more control and flexiblity in
+    // converting types.
+    //
+    // Below, we illustrate three different approaches for the actual conversion
+    // in the closure.
+    // - `From` ensures perfect conversions known at compile time and is the
```

Comment:
I think the official document refers to this as "lossless" instead of "perfect".

Reply:
Updated

Reply:
Seems to be missing?

Reply:
Oh yes, now I understood the comment - thanks! Should be updated now.